mantidproject/mantid
# Indirect Bayes - Quasi tab adjustments

**Only review this PR if you are on windows**

**Description of work.**
This PR makes some minor adjustments to the Indirect Bayes interface including:
- Removal of `Fit` option from Plot Result
- Changes green to blue in fit graph
- `Fit.2` was missing from the `Plot Currrent Preview` plot

**To test:**
1. `Interfaces`->`Indirect`->`Bayes`->`Quasi` Tab
2. Load the data below
3. Click `Run` and wait
4. Make sure there is no greens in the Fit graph (there may be a cyan but that is fine)
5. Click `Plot Current Preview` and make sure there are 5 different lines including one with `fit.2` on the end of its name
6. Look at the plot output combobox and ensure there is no `Fit` option. When you Plot `All` there should be three output graphs corresponding to FWHM, Amplitude and Prob.

**Data Files**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2549444/irs26176_graphite002_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2549445/irs26173_graphite002_res.zip)

Fixes #23829 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
